website/docs/d/network_direct.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_network_direct"
+sidebar_current: "docs-vcd-data-source-network-direct"
+description: |-
+  Provides a vCloud Director Org VDC Network attached to an external one. This can be used to reference internal networks for vApps to connect.
+---
+
+# vcd\_network\_direct
+
+Provides a vCloud Director Org VDC Network data source directly connected to an external network. This can be used to reference
+internal networks for vApps to connect.
+
+Supported in provider *v2.5+*
+
+
+## Example Usage
+
+```hcl
+data "vcd_network_direct" "net" {
+  org  = "my-org"
+  vdc  = "my-vdc"
+  name = "my-net"
+}
+
+# Get the name of the external network from the data source
+# and use it to establish a second data source
+output "external_network" {
+  value = data.vcd_network_direct.net.external_network
+}
+
+# From the second data source we extract the basic networking info
```

Comment:
There isn't datasource `external_network1` in description

Reply:
fixed